dockerfiles/che/entrypoint.sh

```diff
@@ -280,6 +280,18 @@ init() {
   export JAVA_OPTS="${JAVA_OPTS} -Dche.docker.network=$NETWORK_NAME"
 }
 
+add_cert_to_truststore() {
+
+    if [ "${OPENSHIFT_IDENTITY_PROVIDER_CERTIFICATE}" != "" ]; then
+        if [ ! -f /data/openshift.crt ]; then
```

Comment:
If `OPENSHIFT_IDENTITY_PROVIDER_CERTIFICATE` is set and `/data/openshift.crt` already exists then existing file will be added to java store. But there no guarantee that environment variable and file have the same values. Am I correct?

Reply:
Good point. So, there will be problems updating the cert.  I think it's better not to persist it but save elsewhere, so that the file is recreated each time.